src/Widgets/Documents/README.md

```diff
@@ -11,7 +11,7 @@ The module is also extensible, which means that other modules can serve as plug-
 
 ## Basic functionnalities
 
-In the [demo](http://rict2.liris.cnrs.fr/UD-Viz/UD-Viz-Core/src/Modules/Documents/Example/Demo.html), you can try out the basic functionnalities of the document modules. It can:
+In the [demo](http://rict.liris.cnrs.fr/UDVDemo/UDV/UDV-Core/examples/DemoFull/Demo.html), you can try out the basic functionalities of the document modules. It can:
```

Comment:
I would rather point to something local to this repository (like 
`UD-Viz/examples/DocumentWidget`) rather than to an installed demo that might be moved to another deployment server thus breaking this URL link.